.travis.yml

```diff
@@ -3,18 +3,18 @@ _aliases:
   - &tox
     language: python
     python: 3.6
-    before_install: nvm install 8
+    before_install: nvm install 10
     install: pip install tox
     script: tox
   - &node-prod
     # This is the NodeJS version we run in production.
     language: node_js
-    node_js: '8'
+    node_js: '10'
     before_install: npm install -g yarn
   - &node-next
     # This is the next NodeJS version we will support.
     language: node_js
-    node_js: '10'
+    node_js: '11'
```

Comment:
nit: shouldn't this be 12 (since you said we wont be using 11 anyway)?

Reply:
12 has not been released yet, iirc

Reply:
I guess I was going by this comment:

`This is the next NodeJS version we will support.` 

which you mentioned would be 12 above

Reply:
Yep, because LTS versions are even numbers (6, 8, 10, etc.) and odd numbers are not.